ansible/roles/app/files/app_dev.php.dist

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Debug\Debug;
+
+if (isset($_SERVER['HTTP_CLIENT_IP'])
```

Comment:
please remove this check completely, as we can't do a thing with it in place :wink: